modules/administration-guide/pages/configuring-single-and-multiroot-workspaces.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+:description: Configuring single and multiroot workspaces
+:keywords: singleroot, multiroot, workspace
+:navtitle: Configuring single and multiroot workspaces
+// :page-aliases:
+
+[id="configuring-single-and-multiroot-workspaces"]
+= Configuring single and multi root workspaces
+
+With the multi-root workspace feature, you can work with multiple project folders in the same workspace. This is useful when you are working on several related projects at once, such as product documentation and product code repositories.
+
+[NOTE]
+====
+Starting with {prod-short} 3.11, the workspace is set to open in multi-root mode by default.
+====
+
+Once workspace is started, the `</projects/.code-workspace>` workspace file is generated. The workspace file will contain all the projects described in the devfile.
+
+[source,json]
+----
+{
+	"folders": [
+		{
+			"name": "<project-1>",
+			"path": "</projects/project-1>"
```

Comment:
Using the `<` and `>` is confusing (here as well in the rest of the article). A user may consider that as an example and add the `<` and `>` in the code-workspace. Since you are using some example values (`project-1` and `/projects/project-1`) and not a template (`<project-name>` and `<project-path>`) you should remove the `<` and `>`.

Also I think somewhere you should add a link to https://code.visualstudio.com/docs/editor/workspaces as the reference to understand and author VS Code workspace files.

Reply:
done